### PR TITLE
[Project Enhancement]: Brain Stroke Prediction using Neural Networks

### PR DESCRIPTION
**Pull request by Armaan457**
# Pull Request for ML-Crate 💡

Brain Stroke Prediction using Neural Networks #605 

- Do the data analysis and cleaning of the given data and build a neural network with optimum accuracy for the same.
- Armaan Jagirdar
- armaanjagirdar05@gmail.com
- Armaan457
- Contributor at SSOC'24

Closes: #605

Added a new notebook (ipynb) containing the codes for data cleaning and analysis along with 5 neural networks build on TensorFlow with various layers, activations and regularisers

## Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? ⚙️

Describe how it has been tested: The accuracy of the models were measured to ensure no underfitting or overfitting occurs, the data was cleaned to ensure no null values interfere.

## Checklist: ☑️

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
Why have you removed the existing project files?

**Comment by Armaan457** [comment]
I'm sorry sir, I wasn't aware about this. What changes should I do now?

**Comment by abhisheks008** [comment]
> I'm sorry sir, I wasn't aware about this. What changes should I do now?

Put the existing files as it is and add your enhanced models/files along with the existing files.

**Comment by Armaan457** [comment]
Done Sir

**Comment by abhisheks008** [comment]
Follow the README template and update the new README accordingly. Here is the template, https://github.com/abhisheks008/ML-Crate/blob/main/.github/readme_template.md
@Armaan457

**Comment by Armaan457** [comment]
Done sir, kindly review and suggest any changes needed.
The readme in in the Model folder inside the New folder

**Review by abhisheks008 — APPROVED** [review]
Approved @Armaan457
